src/views/Home/components/Question/index.js

```diff
@@ -0,0 +1,217 @@
+                  label={'No'}
+                  onChange={() => {
+                    this.setState({ qOneAnswer: 'No', currentStatus: false });
+                  }}
+                />
+                <FormControlLabel
+                  value="Yes"
+                  control={<Radio />}
+                  label={`Yes`}
+                  onChange={() => {
+                    this.setState({ qOneAnswer: 'Yes', currentStatus: true });
+                  }}
+                />
+              </RadioGroup>
+            </FormControl>
+          </div>
+        </CardContent>
+      );
+    }
+    // Shows the Gordon Loader if the data has not yet been found in the state
+    else {
+      return <GordonLoader />;
+    }
+  }
+
+  // Creates the second question of the wellness check
+  createQuestionTwo(questionStyle) {
+    // Checks to make sure the questions are found in the state before
+    if (this.state.questions !== null) {
+      if (this.state.qOneAnswer === 'Yes') {
+        return (
+          <CardContent>
+            <div style={questionStyle}>
+              <FormControl>
+                <FormLabel>
+                  {this.state.questions.qOne.yes.question[0]}
+                  <a
+                    href=" https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html#cdc-chat-bot-open"
+                    target="_blank"
+                  >
+                    {this.state.questions.qOne.yes.question[1]}
+                  </a>
+                  {this.state.questions.qOne.yes.question[2]}
+                </FormLabel>
+              </FormControl>
+            </div>
+          </CardContent>
+        );
+      } else if (this.state.qOneAnswer === 'No') {
+        return (
+          <CardContent>
+            <div style={questionStyle}>
+              <FormControl>
+                <FormLabel>{this.state.questions.qOne.no.question}</FormLabel>
+              </FormControl>
+            </div>
+          </CardContent>
+        );
+      }
+    }
+  }
+
+  showSubmitButton(buttonStyle) {
+    // Shows submit button
+    if (this.state.questions && this.state.qOneAnswer) {
+      return (
+        <div>
+          <br />
+          <Button variant="contained" style={buttonStyle.button} onClick={this.submitHandler}>
+            Submit
+          </Button>{' '}
+          <br />
+          <br />
+        </div>
+      );
+    }
+    // Doesn't show submit button
+    return <div></div>;
+  }
+
+  render() {
+    // Styles the header
+    const headerStyle = {
+      backgroundColor: gordonColors.primary.blue,
+      color: '#FFF',
+      padding: '10px',
+      fontSize: 20,
+    };
+
+    // Styles each question
+    const questionStyle = {
+      textAlign: 'left',
+    };
+
+    // Styles the submit button
+    const buttonStyle = {
+      button: {
+        background: gordonColors.primary.cyan,
+        color: 'white',
+      },
+    };
+
+    if (this.state.error) {
```

Comment:
I'm not sure why this is necessary.  Can you explain why, or where it came from?

Reply:
seems like boiler plate error catching code that is present on many of the main pages of 360. Included for the sake of following coding pattern.